urdf2webots/parserURDF.py

```diff
@@ -87,6 +88,14 @@ def __init__(self):
         self.radius = 0.0
 
 
+class Mesh():
+    """Define mesh object."""
+
+    def __init__(self):
+        """Initializatization."""
+        self.url = ""
```

Comment:
We prefer single quotes:
```suggestion
        self.url = ''
```